includes/functions/llms.functions.templates.dashboard.php

```diff
@@ -529,6 +603,48 @@ function lifterlms_template_student_dashboard_my_courses( $preview = false ) {
 	}
 }
 
+if ( ! function_exists( 'lifterlms_template_student_dashboard_my_favorites' ) ) {
```

Comment:
`llms_` is the correct prefix.
https://github.com/gocodebox/lifterlms/blob/trunk/docs/coding-standards.md#functions--variables

Reply:
fixed.